ethstaker_deposit/key_handling/key_derivation/mnemonic.py

```diff
@@ -127,7 +128,8 @@ def reconstruct_mnemonic(mnemonic: str, words_path: str) -> Optional[str]:
                 This check guarantees that only one language has a valid mnemonic.
                 It is needed to ensure abbrivated words aren't valid in multiple languages
                 """
-                assert reconstructed_mnemonic is None
+                if reconstructed_mnemonic is not None:
+                    raise ValidationError("This mnemonic abbrivated form is available in multiple languages.")
```

Comment:
Very minor but
```suggestion
                    raise ValidationError("This mnemonic abbreviated form is available in multiple languages.")
```

Looks like the comment was initially wrong as well